### DISCUSSION TITLE
Booting from USB

### DISCUSSION THREAD

**Discussion by ghost** [HotCakeX/Harden-Windows-Security#464] [Q&A]
Hi,

Is there something in your security module that would block a different system from booting from a USB? I was trying to use RedkeyUSB and it would not go into the operating system.

Thank you!

**Comment by HotCakeX** [comment]
Hi,
Set up BitLocker using the Harden Windows Security module, it's the strongest possible protection against preventing other OS boots and doesn't need any 3rd party software.

https://github.com/HotCakeX/Harden-Windows-Security/wiki/Harden%E2%80%90Windows%E2%80%90Security%E2%80%90Module

https://github.com/HotCakeX/Harden-Windows-Security/wiki/BitLocker,-TPM-and-Pluton-%7C--What-Are-They-and-How-Do-They-Work

**Reply by ghost** [reply]
Thank you! Actually I am trying boot from a USB. If run your module how can I do that if the bitlocker category is preventing it?

**Reply by HotCakeX** [reply]
By knowing the same PIN you set for BitLocker startup PIN.
It only prevents people who don't know the PIN. It's like the door in your house, it doesn't mean no one can enter, it means only people with the key can enter.

**Reply by ghost** [reply]
Sorry I should have clarified. This after I have unlocked bitlocker with my pin.

**Reply by HotCakeX** [reply]
After that you can boot to another OS via USB, have to first change boot order in UEFI settings.
